_posts/2017-01-06-conduit-0.2.1.md

```diff
@@ -3,14 +3,12 @@ title: "Conduit 0.2.1 Released"
 tags: new-release
 ---
 
-Version 0.2.1 of Conduit is now available at: https://software.llnl.gov/conduit/releases.html
-
 Conduit provides APIs focused on simplifying data exchange in HPC simulations. It provides an intuitive model for describing hierarchical scientific data in C++, C, Fortran, and Python and is used for data coupling between packages in-core, serialization, and I/O tasks.
 
-The 0.2.1 version is primarily a bug fix releases, which includes:
+The [0.2.1 version](https://llnl-conduit.readthedocs.io/en/latest/) is primarily a bug fix releases, which includes:
```

Comment:
Since the text is specifically about the 0.2.1 version, we may want to link to the docs for that specific version.
https://llnl-conduit.readthedocs.io/en/v0.2.1/

Reply:
Agreed.